core/src/main/java/org/apache/gravitino/storage/relational/mapper/OwnerMetaMapper.java

```diff
@@ -0,0 +1,161 @@
+          + " #{ownerRelPO.metadataObjectId},"
+          + " #{ownerRelPO.metadataObjectType},"
+          + " #{ownerRelPO.ownerId},"
+          + " #{ownerRelPO.ownerType},"
+          + " #{ownerRelPO.auditInfo},"
+          + " #{ownerRelPO.currentVersion},"
+          + " #{ownerRelPO.lastVersion},"
+          + " #{ownerRelPO.deletedAt}"
+          + ")")
+  void insertOwnerEntityRel(@Param("ownerRelPO") OwnerRelPO ownerRelPO);
+
+  @Update(
+      "UPDATE "
+          + OWNER_TABLE_NAME
+          + " SET deleted_at = (UNIX_TIMESTAMP() * 1000.0)"
+          + " + EXTRACT(MICROSECOND FROM CURRENT_TIMESTAMP(3)) / 1000"
+          + " WHERE metadata_object_id = #{metadataObjectId} AND metadata_object_type = #{metadataObjectType} AND deleted_at = 0")
+  void softDeleteOwnerRelByEntityIdAndType(
+      @Param("metadataObjectId") Long metadataObjectId,
+      @Param("metadataObjectType") String metadataObjectType);
+
+  @Update(
+      "UPDATE  "
+          + OWNER_TABLE_NAME
+          + " SET deleted_at = (UNIX_TIMESTAMP() * 1000.0)"
+          + " + EXTRACT(MICROSECOND FROM CURRENT_TIMESTAMP(3)) / 1000"
+          + " WHERE metalake_id = #{metalakeId} AND deleted_at =0")
+  void softDeleteOwnerRelByMetalakeId(@Param("metalakeId") Long metalakeId);
+
+  @Update(
+      "UPDATE  "
+          + OWNER_TABLE_NAME
+          + " ot SET ot.deleted_at = (UNIX_TIMESTAMP() * 1000.0)"
+          + " + EXTRACT(MICROSECOND FROM CURRENT_TIMESTAMP(3)) / 1000"
+          + " WHERE EXISTS ("
+          + " SELECT ct.catalog_id FROM "
+          + CatalogMetaMapper.TABLE_NAME
+          + " ct where ct.catalog_id = #{catalogId}  AND ct.deleted_at = 0 AND ot.deleted_at = 0 AND "
+          + "ct.catalog_id = ot.metadata_object_id AND ot.metadata_object_type = 'CATALOG'"
+          + " UNION "
+          + " SELECT st.catalog_id FROM "
+          + SchemaMetaMapper.TABLE_NAME
+          + " st where st.catalog_id = #{catalogId} AND st.deleted_at = 0 AND ot.deleted_at = 0 AND "
+          + "st.schema_id = ot.metadata_object_id AND ot.metadata_object_type = 'SCHEMA'"
+          + " UNION "
+          + " SELECT tt.catalog_id FROM "
+          + TopicMetaMapper.TABLE_NAME
+          + " tt where tt.catalog_id = #{catalogId} AND tt.deleted_at = 0 AND ot.deleted_at = 0 AND "
+          + "tt.topic_id = ot.metadata_object_id AND ot.metadata_object_type = 'TOPIC'"
+          + " UNION "
+          + " SELECT tat.catalog_id FROM "
+          + TableMetaMapper.TABLE_NAME
+          + " tat where tat.catalog_id = #{catalogId} AND tat.deleted_at = 0 AND ot.deleted_at = 0 AND "
+          + "tat.table_id = ot.metadata_object_id AND ot.metadata_object_type = 'TABLE'"
+          + " UNION "
+          + " SELECT ft.catalog_id FROM "
+          + FilesetMetaMapper.META_TABLE_NAME
+          + " ft where ft.catalog_id = #{catalogId} AND ft.deleted_at = 0 AND ot.deleted_at = 0 AND"
+          + " ft.fileset_id = ot.metadata_object_id AND ot.metadata_object_type = 'FILESET'"
```

Comment:
Change `where` to uppercase, also double check if all the reserved words are uppercase or not.

Reply:
@jerqi  Also have a `where` in the `org/apache/gravitino/storage/relational/mapper/SecurableObjectMapper.java`, Please change it to uppercase.

Reply:
Done.